CONTRIBUTING.md

```diff
@@ -100,15 +79,7 @@ Once your changes are ready to submit for review:
 
 The test suite consists of xUnit tests.
 
-To run all tests, including the integration tests, execute these two commands from the root of the repository:
-
-```bash
-dotnet test test/Elastic.Apm.Tests/
-```
-
-```bash
-dotnet test test/Elastic.Apm.AspNetCore.Tests/
-```
+To run all tests, including the integration tests, execute `dotnet test` on every project from the `/test` folder.
```

Comment:
This was outdated - best is to just say that people should run everything from `/test`.